uxarray/grid.py

```diff
@@ -0,0 +1,279 @@
+"""uxarray grid module."""
+import os
+import xarray as xr
+import numpy as np
+from pathlib import PurePath
+
+
+class Grid:
+    """A class for uxarray grid object.
```

Comment:
Maybe something like "The Uxarray Grid object class that describes an unstructured grid"